src/callbacks/onLayout/initStudyDayControl.js

```diff
@@ -0,0 +1,82 @@
+import { extent } from 'd3';
+import startTransition from './initStudyDayControl/startTransition';
+import stopTransition from './initStudyDayControl/stopTransition';
+
+export default function initStudyDayControl() {
+    var chart = this;
+    var config = this.config;
+    const studyDayControlWrap = this.controls.wrap
+        .selectAll('div')
+        .filter(controlInput => controlInput.label === 'Study Day');
+
+    chart.controls.studyDayInput = studyDayControlWrap.select('input');
+
+    //convert control to a slider
+    chart.controls.studyDayInput.attr('type', 'range');
+
+    //set min and max values and add annotations
+    chart.controls.studyDayRange = extent(chart.imputed_data, d => d[config.studyday_col]);
+    chart.controls.studyDayInput.attr('min', chart.controls.studyDayRange[0]);
+    chart.controls.studyDayInput.attr('max', chart.controls.studyDayRange[1]);
+    chart.controls.studyDayInput.attr('step', 7);
```

Comment:
Not sure I like this attribute because it rounds `config.plot_day` to the nearest multiple of 7, counting from the minimum study day.

Reply:
Yeah, let's get rid of that attribute. We can figure out a better way to handle step size in the animation.